package.json

```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "description": "Gladia documentation",
   "main": "index.js",
+  "author": "",
+  "license": "ISC",
   "scripts": {
+    "mintlify": "mintlify",
```

Comment:
Was that necessary?

Reply:
I suppose to avoid conflict with `-g mintlify`